.gitignore

```diff
@@ -24,3 +24,6 @@ kino-*.tar
 
 # Temporary files, for example, from tests.
 /tmp/
+
+# Node modules.
+lib/assets/*/node_modules/
```

Comment:
My suggestion is to have an `assets` folder at the root and compile the artefacts to `lib/assets`. Otherwise we can accidentally publish a Hex package with the whole copy of `node_modules` and so on. :D